lib/recorder/audio.ts

```diff
@@ -73,10 +73,12 @@ export class AudioRecorder {
 
   /**
    * Generates the object url that can be used for audio playback from raw audio data
-   */
-  private _generateObjectUrl(): void {
-    // Use wav for faster and simple encoding
-    const blob = new Blob(this._audioData, { type: 'audio/wav' });
+  */
+ private _generateObjectUrl(): void {
+    // Select default browser mime type if it exists.
+    // Otherwise, use wav for faster and simple encoding.
+    const type: string = this._mediaRecorder ? this._mediaRecorder.mimeType : 'audio/wav';
```

Comment:
Is `MediaRecorder.mimeType` cross browser? Maybe something like the following?
```ts
const type: string = this._mediaRecorder && this._mediaRecorder.mimeType ? this._mediaRecorder.mimeType : 'audio/wav';
```

Reply:
According to [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/mimeType) it is. I've also been able to access it across various browsers as seen in some of the testing above. But, just in case I'll add the check for it! Thank you!